Source/IncrementableLabel.swift

```diff
@@ -119,6 +129,29 @@ extension IncrementableLabel {
         RunLoop.main.add(timer, forMode: RunLoop.Mode.tracking)
         self.timer = timer
     }
+    
+    //From https://stackoverflow.com/a/39779603
+    private func blend(from: UIColor, to: UIColor, percent: Double) -> UIColor {
```

Comment:
could you move this method in an `UIColor` extension ? (potentially in another swift file)